backup.sh

```diff
@@ -37,12 +37,22 @@ do
   if [ -z "${OPENSSL_ENC_PASS}" ]
   then
     echo "Uploading unencrypted file ${FILE}...";
-    aws s3 cp ${FILE} ${AWS_S3_BUCKET_PATH}${FILE} --storage-class=${AWS_STORAGE_CLASS};
+    if [ -z "${AWS_ENDPOINT_URL}" ]
+    then
+      aws s3 cp ${FILE} ${AWS_S3_BUCKET_PATH}${FILE};
+    else \
+      aws --endpoint-url ${AWS_ENDPOINT_URL} s3 cp ${FILE} ${AWS_S3_BUCKET_PATH}${FILE};
+    fi
```

Comment:
does the `--endpoint-url` already include the `--storage-class`?

Reply:
No, it uses the standard storage class only now but storage class is an Amazon AWS feature only (at least MinIO and Scaleway reject queries when this prop is added), [Amazon recommends to use Standard because Reduced Redundancy is more expensive than Standard](https://docs.aws.amazon.com/AmazonS3/latest/userguide/storage-class-intro.html), other S3 tools [already deprecated this option](https://s3browser.com/working-with-reduced-redundancy-storage-rrs.aspx) so I decided to not add a hack to support Amazon `--storage-class` and for other S3 compatible storage remove it. But if you prefer I can add it again.

Reply:
Scaleway has other storage classes as well, but ok let's not re-add this unless we eventually need it.